src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -202,6 +205,46 @@ export class RoslynLanguageServer {
                 workspace: {
                     configuration: (params) => readConfigurations(params),
                 },
+                resolveCodeAction: async (codeAction, token, next) => {
+                    const lspCodeAction = <CodeAction>codeAction;
+                    const data = lspCodeAction.data;
+                    if (data.FixAllFlavors) {
```

Comment:
I know it comes from Roslyn, but it is amusing that it is called FixAll"Flavors" and not "Scopes" or "Kinds".